src/index.ts

```diff
@@ -0,0 +1,74 @@
+// tslint:disable:no-console
+import {
+  Beautifier,
+  Language,
+  BeautifierBeautifyData,
+  DependencyType,
+  NodeDependency,
+} from "unibeautify";
+import * as cosmiconfig from "cosmiconfig";
+import * as readPkgUp from "read-pkg-up";
+import options from "./options";
+
+const { pkg } = readPkgUp.sync({ cwd: __dirname });
+export const beautifier: Beautifier = {
+  name: "CSScomb",
+  package: pkg,
+  dependencies: [
+    {
+      type: DependencyType.Node,
+      name: "CSScomb",
+      package: "csscomb",
+    },
+  ],
+  options: {
+    CSS: options.CSS,
+    SCSS: options.CSS,
+    Sass: options.CSS,
+  },
+  resolveConfig: ({ filePath, projectPath, dependencies }) => {
+    if (!filePath) {
+      return Promise.resolve({});
+    }
+    const Comb = dependencies.get<NodeDependency>("CSScomb").package;
+    const explorer = cosmiconfig("csscomb", {
+      searchPlaces: [
+        ".csscomb.json",
+        "csscomb.json",
+        ".csscomb.js",
+        "csscomb.js",
+      ],
+    });
+    const cosmicFund = explorer.searchSync(filePath);
```

Comment:
Do not need to use the `sync` method, if there is an async method you can return a promise 👍 .